### ISSUE TITLE
Orphaned EXCEL32 and pythonw processes after exiting Excel when using the optimized connection

### ISSUE THREAD

**Issue by acommendador** [ZoomerAnalytics/xlwings#193]
While using xlwings 0.3.5 with the xlwings.bas added to an Excel add-in and the optimized connection enabled, exiting Excel with the pythonw process started closes the Excel GUI but leaves the EXCEL32 and pythonw processes running in the background.  We've tried the following from a python function with no success: invoking the active workbook application quit method, and invoking <code>sys.exit</code>.  We've also tried replacing all instances of <code>ThisWorkbook</code> with <code>ActiveWorkbook</code> within the xlwings.bas.

**Comment by acommendador** [comment]
After some investigation we've found that the problem only occurs when getting a handle to the Workbook object within python.  We also found that the the xlwings/server.py serve method never exits when Excel is closed.  We were able to force the serve method to return when Excel closes by adding a python method to invoke <code>ctypes.windll.user32.PostQuitMessage(0)</code> (see http://stackoverflow.com/questions/6023172/ending-a-program-mid-run) and by invoking the method via a VBA module method named auto_close.
